Package.swift

```diff
@@ -62,25 +62,42 @@ let package = Package(
     ]
 )
 
-func loadSwiftLintVersion() -> Version {
+func swiftLintPlugin() -> PackageDescription.Package.Dependency {
+    let url = "https://github.com/realm/SwiftLint"
+
+    // Swift can access and read the config file with the current logic only within the package repo.
+    // This is, the check will fail when 'swift build' attempts to resolve the package as a dependency.
+    // To workaround this limitation, we fall back to the latest version of the plugin when the config file read fails.
+    // This should be safe to do, given the SwiftLint plugin is only used in development.
+    guard let version = loadSwiftLintVersion() else {
+        return .package(url: url, from: "0.54.0")
+    }
+
+    return .package(url: url, exact: version)
+}
+
+func loadSwiftLintVersion() -> Version? {
     let swiftLintConfigURL = URL(fileURLWithPath: #file)
```

Comment:
> the check will fail when 'swift build' attempts to resolve the package as a dependency.

Have you tried using `#filePath` here?

Reply:
I'll give it a shot. But, I don't expect it to make much difference.

According to [the docs](https://developer.apple.com/documentation/swift/file()):

> Currently, `#file` has the same value as `#filePath`.

Reply:
Tried via `mokagio/swiftlint-file-path` branch. Same result 😞